apps/cowswap-frontend/index.html

```diff
@@ -8,7 +8,6 @@
       name="description"
       content="CoW Swap finds the lowest prices from all decentralized exchanges and DEX aggregators & saves you more with p2p trading and protection from MEV"
     />
-    <base href="/"/>
```

Comment:
Any clue why this was here?

Reply:
No idea, but drove me crazy. I hoped you or Sasha would know